agent_runtime_profile/.claude/agents/analyze-assets.md

```
@@ -56,17 +56,25 @@ description: 从剧本中提取角色 / 场景 / 道具三类资产定义，按
 - 提取重复出现或具有视觉特征的物品/道具
 - description 包含：外观细节、材质、尺寸参考、色彩特征
 
-### Step 4: 调用脚本写入 project.json
+### Step 4: 调用工具写入 project.json
 
-⚠️ 必须单行，JSON 使用紧凑格式，不可用 `\` 换行：
+每个资产表（characters / scenes / props）调用一次 `mcp__arcreel__patch_project`，按 name upsert：
 
-```bash
-python .claude/skills/manage-project/scripts/add_assets.py --characters '{"角色名1": {"description": "视觉描述...", "voice_style": "声音风格..."}, "角色名2": {"description": "视觉描述...", "voice_style": "声音风格..."}}' --scenes '{"庙宇": {"description": "空间描述..."}, "客栈大堂": {"description": "环境描述..."}}' --props '{"玉佩": {"description": "外观描述..."}, "长剑": {"description": "外观描述..."}}'
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)